app/services/updaters/group_updater.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# Service object to update groups
```

Comment:
I think we're somewhat inconsistent about this across the board; I think that the empty line is probably better so I'll add it